src/components/ProviderSettings/connect.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import { createStore } from '../../store';
+import connect, { mapStateToProps, mapDispatchToProps } from './connect';
+import providerModel, * as providerActionCreators from '../../store/models/provider';
+
+jest.mock('../../store/models/provider');
+
+describe('mapStateToProps(state, props)', () => {
+  let isLoading = jest.fn(() => 'test isLoading');
+  let getError = jest.fn(() => 'test getError');
+  let getCurrentProvider = jest.fn(() => 'test getCurrentProvider');
+  providerModel.mockReturnValue({ isLoading, getError, getCurrentProvider });
+});
```

Comment:
I guess your `describe` should be wrapping the below test case. I would also suggest initializing the mock function within `beforeEach` so in the future you don't need to worry the previous test case may affect the result of newer test cases.

Reply:
Agreed